CONTRIBUTING.md

```diff
@@ -1,29 +1,22 @@
 
-The branch `1.x` contains Web Components V0 & Polymer 1 client-side libraries intended as:
+The name of the StarcounterClientFiles branch used by the Level1 build is configured via the `GIT_CLIENTFILES_BRANCH` parameter in TeamCity.
 
-- the default and only client-side libraries supported in Starcounter 2.3
-- the way to downgrade Starcounter 2.4 to WCv0 & P1)
 
-There is also another branch, `2.x`, which contains Web Components V1 and Polymer 2.
+## Releasing of a new version of StarcounterClientFiles 1.x
 
-## Development
-
-Currently, the developer process looks like follows:
+The developer process of releasing of a new version of StarcounterClientFiles `1.x` to Starcounter:
 
 1. Commit changes to individual client-side package's repo, like https://github.com/Starcounter/starcounter-include, and release a version there,
-2. Bump files in a feature branch started from `1.x` in [this repo](https://github.com/Starcounter/StarcounterClientFiles) either,
+2. Bump files in a feature branch of StarcounterClientFiles started from `1.x`, either:
    - by updating `bower.json` and running `bower install`,
-   - or `calling `bower install --save packagename[#version]`
+   - or by calling `bower install --save packagename[#version]`
 3. Update `bower-list.txt` by calling `bower list > bower-list.txt`
-4. Make a PR for someone to review the complete set of changes.
-5. As soon as the PR is merged, release new version (1.x) of StarcounterClientFiles (remember to write the release notes)
-6. Publish a package of this version to App Warehouse (built with Starcounter 2.4)
-7. Copy manually to a custom branch at level1 (forked from `v2.3`)
-8. Build level1 branch in TeamCity
-9. Build KitchenSink, Website and CompositionProvider with custom level1 build (pick 2.3 task)
-10. If all tests pass, create a PR to level1 (remember to bump `CHANGELOG.md`)
-11. You don't need a reviewer for this PR. Simply do something similar to what @warpech did in https://github.com/Starcounter/level1/pull/4420 and give the link to the PR in StarcounterClientFiles where it was reviewed in more detail.
-12. Merge the PR to `v2.3` and then merge `v2.3` to `develop`. Take proper steps to ensure that no unintended files were overwritten in `develop`. If yes, solve the conflict.
+4. When you push to a feature branch of StarcounterClientFiles, TeamCity will run a integration test of this branch with KitchenSink, WebsiteProvider and CompositionProvider. Make sure these tests are passing.
+5. Make a PR for someone to review the complete set of changes.
+6. Still in the feature branch, release new version (1.x) of StarcounterClientFiles on GitHub. Remember to write the release notes.
+7. Still in the feature branch, publish a package of this version to App Warehouse (built with Starcounter 2.3)
+8. Merge the PR to StarcounterClientFiles branch `1.x`.
+   - at this point, a new build of Level1 is started
```

Comment:
I am uncertain if this specific part is true.

@joozek78 Isn't it only `Level0` that has the automatic trigger on PR:s?

Reply:
Actually, I want to know if either of the following is true:

1. a new build of Level1 is started when a commit is pushed to StarcounterClientFiles `1.x` / `2.x` branch
2. or: a nightly build of Level1 will include changes from pushed to StarcounterClientFiles `1.x` / `2.x` even if no changes were detected in the Level1 repo

Reply:
1. Yes, I belive it is, if you mean `daily windows build` by `Level1 build`. https://github.com/Starcounter/AdminTrack/issues/438#issuecomment-363460042
2. The `nightly windows builds` are only triggered at night if there are changes in any of the 11 repositories "default" branch where `StarcounterClientFiles` is one of them. So this step will not be a problem for you.

Reply:
Perfetto!

Reply:
Added a clarification about that